K2Engineering/K2Engineering/CastBarOutput.cs

```diff
@@ -45,17 +45,19 @@ protected override void RegisterOutputParams(GH_Component.GH_OutputParamManager
         protected override void SolveInstance(IGH_DataAccess DA)
         {
             //Input
-            Object[] output = new Object[4];
+
+            //Object[] output = new Object[4];
```

Comment:
Leave the code versioning to revision control (in this case git) :) commented out code is a code smell